### PR TITLE
Rewrite pth files as well

### PR DESCRIPTION
**Pull request by asottile**
So this makes a couple assumptions, and will unfortunately do the wrong thing without those satisfied.  These assumptions are pretty reasonable imo?

- A relative path computed from the pth file to the path in the pth file's line will be valid after the virtualenv moves.

Consider the following:

```
/app/
 |
 +--- setup.py
 +--- something.py
 +--- venv/lib/site-packages/easy-install.pth: contents '/app\n'
```

- **valid** : `mv /app /newapp && virtualenv_tools --update-path=/newapp/venv /newapp/venv`
- **invalid**: `mv /app/venv /venv && virtualenv_tools --update-path=/venv /venv`

That is, in the first, the relative path from the pth file to the editable installed app is the same before and after because the app is moving with the virtualenv.  If the editable target does not move with the application the paths will be incorrect

Thoughts? Any cases I'm missing?

### CODE REVIEW

File: tests/virtualenv_tools_test.py [hunk 1]
```diff
@@ -63,25 +76,48 @@ def test_each_part_idempotent(tmpdir, venv, capsys):
     assert out == expected
 
 
+def _assert_activated_sys_executable(path):
+    exe = subprocess.check_output((
+        'bash', '-c',
+        ". {} && python -c 'import sys; print(sys.executable)'".format(
```
Comment (bukzor) [inline tests/virtualenv_tools_test.py:82]: only needs to be `command -v python` imo
obviously not a regression

File: virtualenv_tools.py [hunk 2]
```diff
@@ -167,6 +167,33 @@ def get_new_path(filename):
...
+    changed = False
+    for i, line in enumerate(lines):
+        val = line.strip()
+        if val.startswith('import ') or not os.path.isabs(val):
+            continue
+        changed = True
+        relto_original = os.path.relpath(val, orig_path)
+        # lib/pythonX.X/site-packages
+        relto_pth = os.path.join('../../..', relto_original)
+        lines[i] = '{}\n'.format(relto_pth)
```
Comment (bukzor) [inline virtualenv_tools.py:182]: I'd rather see:

```python
if val.startswith('import '):
    continue

orig_val = val
val = os.path.abspath(val, site-packages)
val = os.path.relpath(val, site-packages)
changed |= orig_val == val
```

That way you don't need to hard-code the fact that site-packages is three directories deep, which seems more foolproof to me, albeit quite unlikely to give different behavior.

**Review by bukzor — COMMENTED** [review]
p3 issue: fix or punt

File: virtualenv_tools.py (hunk 2 above)
Comment (asottile) [reply virtualenv_tools.py:182]: So I initially did this, but the original path can be both an abspath and not an abspath and it was unclear a correct way to normalize it that made it correct :S

**Pull request merged**